src/main/resources/l10n/JabRef_tr.properties

```diff
@@ -1684,3 +1684,52 @@ You_must_enter_at_least_one_field_name=En_az_bir_alan_adı_girmelisiniz
 
 
 Non-ASCII_encoded_character_found=ASCII_koduyla_kodlanmamış_karakter_bulundu
+Connection=
+Host=
+Database=
+User=
+Connection_error=
+Driver_error=
+Connection_to_%0_server_stablished.=
+Required_field_"%0"_is_empty.=
+Corrupt_shared_database_structure.=
+Integrity_check_failed._Fixing...=
+
+%0_driver_not_available.=
+
+The_connection_to_the_server_has_been_determinated.=
+
+Connection_lost.=
+Reconnect=
+Work_offline=
+
+Working_offline.=
+
+Update_refused.=
+Update_refused=
+Local_entry=
+Shared_entry=
+Update_could_not_be_performed_due_to_existing_change_conflicts.=
+You_are_not_working_on_the_newest_version_of_BibEntry.=
+Local_version\:_%0=
+Shared_version\:_%0=
+Please_merge_the_shared_entry_with_yours_and_press_"Merge_entries"_to_resolve_this_problem.=
+Canceling_this_operation_will_leave_your_changes_unsynchronized._Cancel_anyway?=
+The_BibEntry_you_currently_work_on_has_been_deleted_on_the_shared_side._Hit_"Keep"_to_recover_the_entry.=
+Cannot_cite_entries_without_BibTeX_keys._Generate_keys_now?=
+New_technical_report=
+
+%0_file=
+Custom_layout_file=
+Protected_terms_file=
+Style_file=
+
+Open_OpenOffice/LibreOffice_connection=
+
+You_must_enter_at_least_one_field_name=
+
+
+Non-ASCII_encoded_character_found=
+
+Background_color_for_resolved_fields=
```

Comment:
I would just add the lines of the language strings you added. The other lines should come in in different ways (other PR, via master branch). Otherwise, we will have duplicate lines (refs #1719)